testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -158,7 +165,7 @@ Any project that has the following Gems enabled:
-| Creating an entity with PhysX Character Controller component    | <ol><li>Create an entity with PhysX Character Controller for each of the [PhysX Character Controller shape(s)](#physx-character-controller-shapes), add a mesh component to it for visibility.<li>Create a PhysX Rigid body entity with mesh added for visibility, set Initial Linear velocity in PhysX Rigid Body component to value other than **0** and set up a scene in which it collides with PhysX Character Controller in Editor Game Mode or in the Game Launcher.</ul></li><li>Assign PhysX Character Controller with various Physics Materials and collide it with PhysX Rigid bodies.</li></ol> | <ul><li>PhysX character controller as a kinematic controller is not affected by gravity and outside forces but can still collide with other PhysX entities.</li><li>PhysX character controller can be assigned with all available shapes and these shapes dictate how other entities collide with PhysX Character Controller.</li><li>Shapes assigned to PhysX Character Controller can be modified with options found in the component.</li></ul>
+| Creating an entity with PhysX Character Controller component    | <ol><li>Create an entity with PhysX Character Controller for each of the [PhysX Character Controller shape(s)](#physx-character-controller-shapes), add a mesh component to it for visibility.<li>Create an entity with a simulated rigid body and a mesh (for visibility, set Initial Linear velocity in PhysX Dynamic Rigid Body component to value other than **0** and set up a scene in which it collides with PhysX Character Controller in Editor Game Mode or in the Game Launcher.</ul></li><li>Assign PhysX Character Controller with various Physics Materials and collide it with simulated rigid bodies.</li></ol> | <ul><li>PhysX character controller as a kinematic controller is not affected by gravity and outside forces but can still collide with other PhysX entities.</li><li>PhysX character controller can be assigned with all available shapes and these shapes dictate how other entities collide with PhysX Character Controller.</li><li>Shapes assigned to PhysX Character Controller can be modified with options found in the component.</li></ul>
```

Comment:
For some of these longer requests can you break it down into a short set of steps?

Reply:
It has only 3 steps and likely written like that on purpose to use the variants at the top of the page to cover more ground without having to create different entries in the table and keep it short. This PR is not meant to change these steps, just use the right terminology about rigid bodies (static, kinematic and simulated).